src/xcode/ENA/ENA/Source/Services/Risk/KeyPackageDownload/KeyPackageDownload.swift

```diff
@@ -119,6 +121,14 @@ class KeyPackageDownload: KeyPackageDownloadProtocol {
 		}
 	}
 
+	func markPackagesAsCheckedForExposures(_ fingerprints: [String]) {
+		do {
+			try downloadedPackagesStore.markPackagesAsCheckedForExposures(fingerprints)
```

Comment:
Would it make sense to pass the store to the ExposureDetection? Then this func would not be needed.

Reply:
I made the requested changed